pkg/gitlab_events/helpers.go

```diff
@@ -1,110 +1,45 @@
 package gitlab_events
 
-import (
-	"encoding/json"
-	"strings"
-)
-
-// refKind defined what kind of ref as generated an event (branch, tag, merge_request, ...)
-type refKind string
+// Kind defines the kind of ref that generated an event (branch, tag, merge_request, ...)
+type Kind string
```

Comment:
Type `Kind` should be exported as it is returned by a function exported by this package